src/commands/api-mesh/set-log-forwarding.js

```diff
@@ -0,0 +1,151 @@
+		} catch (error) {
+			this.error(error.message);
+			return;
+		}
+		const { imsOrgCode, projectId, workspaceId, workspaceName } = await initSdk({
+			ignoreCache,
+		});
+
+		let meshId = null;
+
+		try {
+			meshId = await getMeshId(imsOrgCode, projectId, workspaceId, workspaceName);
+		} catch (err) {
+			this.log(err.message);
+			this.error(
+				`Unable to get mesh ID. Check the details and try again. RequestId: ${global.requestId}`,
+			);
+		}
+
+		// mesh could not be found
+		if (!meshId) {
+			this.error(
+				`Unable to get meshId. No mesh found for Org(${imsOrgCode}) -> Project(${projectId}) -> Workspace(${workspaceId}). Check the details and try again.`,
+			);
+		}
+
+		let shouldContinue = true;
+
+		if (!autoConfirmAction) {
+			shouldContinue = await promptConfirm(
+				`Are you sure you want to set log forwarding to ${destinationConfig.destination}?`,
+			);
+		}
+
+		if (shouldContinue) {
+			try {
+				const response = await setLogForwarding(
+					imsOrgCode,
+					projectId,
+					workspaceId,
+					meshId,
+					destinationConfig,
+				);
+				if (response && response.result) {
+					this.log('Log forwarding successfully.');
+					return { destinationConfig, imsOrgCode, projectId, workspaceId, workspaceName };
+				} else {
+					this.error(
+						`Unable to set log forwarding details. Please try again. RequestId: ${global.requestId}`,
+					);
+					return;
+				}
+			} catch (error) {
+				this.log(error.message);
+				this.error(
+					`Failed to set log forwarding details. Try again. RequestId: ${global.requestId}`,
+				);
+			}
+		} else {
+			this.log('log-forwarding cancelled');
+			return 'set-log-forwarding cancelled';
+		}
+	}
+
+	async inputAndValidateConfigs(destinations) {
+		// Prompt for destination
+		const destinationKey = await promptSelect(
+			'Select log forwarding destination:',
+			Object.keys(destinations),
+		);
+		if (!destinationKey) {
+			throw new Error('Destination is required');
+		}
+
+		const destinationConfig = destinations[destinationKey];
+		const inputs = {};
+
+		// For each input defined in the destination config, prompt and validate
+		for (const inputConfig of destinationConfig.inputs) {
+			// Prompt for input value (regular or secret based on config)
+			const promptFn = inputConfig.isSecret ? promptInputSecret : promptInput;
+			const value = await promptFn(inputConfig.promptMessage);
+
+			// Validate the input
+			if (inputConfig.validate) {
+				inputConfig.validate(value);
+			}
+
+			// Store the validated input
+			inputs[inputConfig.name] = value;
+		}
+
+		return {
+			destination: destinationConfig.name,
+			config: inputs,
+		};
+	}
+}
+
+SetLogForwardingCommand.description = `Sets the log forwarding destination for API mesh. 
+- Select a log forwarding destination: Choose from available options ( example : newrelic).
+- Enter the base URI: Provide the URI for the log forwarding service. Ensure it includes the protocol ( example : \`https://www.adobe.com\`).
```

Comment:
Instead of this sample base URI, we can say something like:" If the hosted region of New Relic account is U.S, the base URI can be https://log-api.newrelic.com/log/v1